src/video-interface.js

```diff
@@ -105,10 +105,82 @@ export class VideoInterface {
    *
    * @param {string} unusedMethod
    * @param {function(!./service/action-impl.ActionInvocation)} unusedHandler
-   * @param {ActionTrust} minTrust
+   * @param {ActionTrust} unusedMinTrust
    * @public
    */
-  registerAction(unusedMethod, unusedHandler, minTrust) {}
+  registerAction(unusedMethod, unusedHandler, unusedMinTrust) {}
+}
+
+
+/**
+ * @interface
+ */
+export class TrackingVideoInterface extends VideoInterface {
+  /**
+   * Should return true.
+   * @return {boolean}
+   */
+  isTrackingVideo() {}
```

Comment:
I included this so the code wouldn't be as duck-typing-y and it was the best I could come up with at the moment. I'm open to alternatives